cmd/registry/cmd/upload-manifest.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"log"
+	"context"
+	"github.com/spf13/cobra"
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/cmd/registry/controller"
+	"github.com/apigee/registry/rpc"
+	"google.golang.org/protobuf/proto"
+	"github.com/apigee/registry/cmd/registry/core"
+
+)
+
+func init() {
+	uploadCmd.AddCommand(uploadManifestCmd)
+	uploadManifestCmd.Flags().String("project", "", "ProjectID this manifest should be associated with.")
+}
+
+var uploadManifestCmd = &cobra.Command{
+	Use:   "manifest",
```

Comment:
We can add more information to the usage message to indicate that manifest_path is a required argument. See https://github.com/apigee/registry/blob/bf3525dcc6b7637453ca28035f2a9f1fc85e6d36/cmd/registry/cmd/label.go#L41 and https://github.com/apigee/registry/blob/bf3525dcc6b7637453ca28035f2a9f1fc85e6d36/cmd/registry/cmd/upload-csv.go#L44 for examples (also note that we have some style inconsistencies to resolve)

Reply:
Done.